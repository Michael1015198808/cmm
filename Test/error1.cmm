int inc() {
    1
}
int k = ;
