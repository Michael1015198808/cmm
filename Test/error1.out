Error type B at Line 3: missing ";"
Error type B at Line 4: missing ";"
